src/components/DlChart/DlChartLabels.vue

```diff
@@ -19,16 +19,48 @@
                     :data-index="index"
                     :color="labelColor"
                 >
-                    {{ label }}
+                    {{ label.title || label }}
                 </dl-typography>
                 <dl-typography
                     v-else
-                    :ref="(el) => forwardChildEl(el, index)"
+                    :ref="(el) => forwardChildEl(el, 'textRef', index)"
                     :data-index="index"
                     :size="fontSize"
                     :color="labelColor"
                 >
-                    {{ label }}
+                    {{ label.title || label }}
+                </dl-typography>
+            </div>
+        </div>
+        <div
+            v-if="hasSubtitles"
+            class="dl-chart-labels"
+        >
+            <div
+                v-for="(label, index) in labels"
+                :key="index"
+                class="dl-chart-labels--label"
+            >
+                <dl-tooltip v-if="isOverFlowingSubtitles[index]">
+                    {{ label.subtitle || '' }}
```

Comment:
@SorinOpt wouldnt this have an empty tooltip then ? if lable.subtitle is not assigned ?

Reply:
no, because the tooltip has an ellipsis check, if the string is empty, then the tooltip is not triggered